src/kOS/Module/kOSProcessor.cs

```diff
@@ -75,6 +75,19 @@ public string Tag
         [KSPField(isPersistant = true, guiActive = false)]
         public int MaxPartId = 100;
 
+        // This represents how much EC to consume per executed instruction.
+        // This would be the "variable" component of the processor's power.
+        // TODO: This value needs to be changed to a non-zero number and balanced.
```

Comment:
That's exactly what I was thinking.  This is just a default value.  I'm not going to make a whole commit for just this, but I will add a documentation note.